templates/main.php

```diff
@@ -36,7 +36,8 @@
 			</div>
 			<div id="app-settings-content">
 				<addressBookList></addressBookList>
-				<input type="file" id="contact-import" class="hidden-visually" contactimport /><label for="contact-import">Import</label>
+				<input type="file" id="contact-import" class="hidden-visually" contactimport />
+				<label for="contact-import" class="button" id="upload">Import<span class="icon-upload"></span></label>
```

Comment:
Did you saw my patch?
i didn't pushed it yet because I'm waiting for this PR to be merged: https://github.com/skjnldsv/contacts/tree/skjnldsv-patch-new-translation

Btw, i don't really know how else can i do instead of waiting for this pr? Is there a way for me to make a second PR dependant of a previous one?